libnetwork/drivers/remote/driver_test.go

```diff
@@ -472,14 +472,16 @@ func TestRemoteDriver(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	data := discoverapi.NodeDiscoveryData{
-		Address: "192.168.1.1",
-	}
-	if err = d.DiscoverNew(discoverapi.NodeDiscovery, data); err != nil {
-		t.Fatal(err)
-	}
-	if err = d.DiscoverDelete(discoverapi.NodeDiscovery, data); err != nil {
-		t.Fatal(err)
+	if dr, ok := d.(discoverapi.Discover); ok {
```

Comment:
This change can be dropped once https://github.com/moby/moby/pull/46101 is merged

Reply:
done 👍